staticd/static_vty.c

```diff
@@ -1205,10 +1205,9 @@ static void static_cmgd_bcknd_client_connect(
 }
 
 static cmgd_bcknd_client_params_t cmgd_params = {
-	.name = "Staticd",
+	.name = "staticd",
```

Comment:
Why change this?

Reply:
As discussed to keep it consistent across cmgd and modules.